src/main/java/com/ecore/roles/web/UsersApi.java

```diff
@@ -0,0 +1,14 @@
+package com.ecore.roles.web;
+
+import com.ecore.roles.web.dto.UserDto;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface UsersApi {
+
+    ResponseEntity<List<UserDto>> getUsers();
+
+    ResponseEntity<UserDto> getUser(UUID userId);
```

Comment:
It could be just called "get" or "getById", because it's inside of the users api.